.github/workflows/node.js.yml

```diff
@@ -28,6 +28,7 @@ jobs:
     - run: npm ci
     - run: npm run build --if-present
     - run: npm test
+    - run: npm run coverage
```

Comment:
I think we should setup a new workflow for coverage because:
1. Independent of tests passing
2. Only needs to be done once. Currently we're doing it 3 times.
3. We can setup minimum coverage criteria and enforce the coverage workflow to pass/fail based on that.

Reply:
Yes I am working on the threshold. I will be sending the next PR for that only, this was just for the workflow.

For the independent coverage workflow, should I have a separate VM instance to run coverage on? Or use the [Github Action](https://github.com/codecov/codecov-action) for codecov ?

Reply:
Why not just create another job (similar to `build-ubuntu`) to lint? It'll just install dependencies and lint. :)